src/rust/ensogl/src/display/shape/primitive/def/var.rs

```diff
@@ -411,3 +412,134 @@ define_shape_data_string_operator! { Add add (+) }
 define_shape_data_string_operator! { Sub sub (-) }
 define_shape_data_string_operator! { Mul mul (*) }
 define_shape_data_string_operator! { Div div (/) }
+
+
+
+// ===============================
+// === Trigonometric Functions ===
+// ===============================
+
+impl<T> Sin for Var<T>
+where T: Sin<Output=T> {
+    type Output = Var<T>;
+    fn sin(&self) -> Self {
+        match self {
+            Self::Static  (t) => Var::Static(t.sin()),
+            Self::Dynamic (t) => Var::Dynamic(format!("sin({})",t).into())
+        }
+    }
+}
+
+impl<T> Asin for Var<T>
+where T: Asin<Output=T> {
+    type Output = Var<T>;
+    fn asin(&self) -> Self {
+        match self {
+            Self::Static  (t) => Var::Static(t.asin()),
+            Self::Dynamic (t) => Var::Dynamic(format!("asin({})",t).into())
+        }
+    }
+}
+
+
+impl<T> Cos for Var<T>
+where T: Cos<Output=T> {
+    type Output = Var<T>;
+    fn cos(&self) -> Self {
+        match self {
+            Self::Static  (t) => Var::Static(t.cos()),
+            Self::Dynamic (t) => Var::Dynamic(format!("cos({})",t).into())
+        }
+    }
+}
+
+impl<T> Acos for Var<T>
+    where T: Acos<Output=T> {
+    type Output = Var<T>;
+    fn acos(&self) -> Self {
+        match self {
+            Self::Static  (t) => Var::Static(t.acos()),
+            Self::Dynamic (t) => Var::Dynamic(format!("acos({})",t).into())
+        }
+    }
+}
+
+
+
+// ===================
+// === Square Root ===
+// ===================
+
+impl<T> Sqrt for Var<T>
+where T: Sqrt<Output=T> {
+
+    type Output = Var<T>;
+
+    fn sqrt(&self) -> Self {
```

Comment:
I asked in a comment to previous PR to remove the spacing in simple above - they are removed, but here the spacing is still present. Could you please apply the changes here and everywhere else too? :)